dipy/io/utils.py

```diff
@@ -136,6 +138,80 @@ def decfa_to_float(img_orig):
     return Nifti1Image(out_data, affine=img_orig.affine, header=new_hdr)
 
 
+def is_reference_info_valid(affine, dimensions, voxel_sizes, voxel_order):
+    """ Will validate basic data type and value of spatial attribute.
+    Does not ensure that voxel_sizes and voxel_order are self-coherent with
+    affine.
```

Comment:
This is a bit confusing. Maybe best to list out what exactly this checks? Or provide a few examples in the docstring?